dockers/docker-orchagent/ipinip.conf.j2

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "TUNNEL_DECAP_TABLE:IPINIP_TUNNEL" : {
+            "tunnel_type":"IPINIP",
+            "src_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
+            "dst_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
+            "dscp_mode":"uniform",
+            "ecn_mode":"standard",
```

Comment:
by default this should be copy_from_outer, can you check the asic_type and generate corresponding configuration based on the asic_type?